omnibus/config/templates/datadog-agent/systemd.service.erb

```diff
@@ -8,6 +8,7 @@ StartLimitBurst=5
 [Service]
 Type=simple
 PIDFile=<%= install_dir %>/run/agent.pid
+Environment=HOST_PROC=/proc
```

Comment:
I'm 👎 about adding this to the service file 🤔

Even if customers want to pass environment variable through service files (which I'm not sure is a great idea, unless they don't want the env vars to be available when running Agent commands), they should do so in "override" configuration files, not directly in the service file provided by the package.

Reply:
The idea wasn't to override the environment variable here, it was to provide it in the systemd file as a form of implicit documentation.